GNUmakefile

```diff
@@ -8,43 +9,39 @@ EXCLUDED_PACKAGES := \
 
 PACKAGES := $(shell go list ./... | grep -Fvx -f <(printf '%s\n' $(EXCLUDED_PACKAGES)))
 
-default: fmt lint install generate
+.PHONY: bin-deps clean fmt lint install generate build test snapshot release
+default: clean fmt lint install generate
 
-build:
-	go build -v ./...
+bin-deps:
+	go install go.uber.org/mock/mockgen@latest
```

Comment:
Go supports `tool` directives now, so I think we can avoid this?

See https://www.bytesizego.com/blog/go-124-tool-directive

Reply:
Make sense. Actually in the `go.mod` 1.23.7 is set. I'm going to update golang version and introduce the `tool`.